osgtest/tests/test_29_slurm.py

```diff
@@ -52,6 +69,20 @@ def test_01_slurm_config(self):
                     SLURM_CONFIG % {'short_hostname': SHORT_HOSTNAME, 'cluster': CLUSTER_NAME, 'ctld_log': CTLD_LOG},
                     owner='slurm',
                     chmod=0644)
+        core.config['cgroup.config'] = '/etc/slurm/cgroup.conf'
+        config = SLURM_CGROUPS_CONFIG
+        if core.el_release() == 6:
+            config += "\nCgroupMountpoint=/cgroup"
+        files.write(core.config['cgroup.config'],
+                    SLURM_CGROUPS_CONFIG,
```

Comment:
It looks like this should be `config` instead of `SLURM_CGROUPS_CONFIG`, otherwise the EL6 if block above doesn't really do anything

Reply:
yeah, that's right, will update